### ISSUE TITLE
Fail to use in Python 3.8.7. 

### ISSUE THREAD

**Issue by binggu56** [lukelbd/proplot#242]
### Description

Fail to use in Python 3.8.7. 

### Steps to reproduce

```python
>>> import proplot as plt
>>> plt.subplots()

**Actual behavior**: 

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.8/site-packages/proplot/ui.py", line 506, in subplots
    axs[idx] = fig.add_subplot(
  File "/usr/local/lib/python3.8/site-packages/proplot/figure.py", line 1047, in add_subplot
    return super().add_subplot(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/matplotlib/figure.py", line 1402, in add_subplot
    ax = subplot_class_factory(projection_class)(self, *args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/matplotlib/axes/_subplots.py", line 42, in __init__
    self._axes_class.__init__(self, fig, self.figbox, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/proplot/axes/cartesian.py", line 214, in __init__
    super().__init__(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/proplot/axes/base.py", line 294, in __init__
    self.format(rc_mode=1)  # mode == 1 applies the custom proplot params
  File "/usr/local/lib/python3.8/site-packages/proplot/axes/cartesian.py", line 965, in format
    axis.set_tick_params(which=which, **kwgrid, **kwticks)
  File "/usr/local/lib/python3.8/site-packages/matplotlib/axis.py", line 843, in set_tick_params
    tick._apply_params(**kwtrans)
  File "/usr/local/lib/python3.8/site-packages/matplotlib/axis.py", line 394, in _apply_params
    self.gridline.set(**grid_kw)
  File "/usr/local/lib/python3.8/site-packages/matplotlib/artist.py", line 1113, in set
    return self.update(kwargs)
  File "/usr/local/lib/python3.8/site-packages/matplotlib/artist.py", line 998, in update
    ret.append(func(v))
  File "/usr/local/lib/python3.8/site-packages/matplotlib/artist.py", line 930, in set_alpha
    raise TypeError('alpha must be a float or None')
TypeError: alpha must be a float or None


### Proplot version

0.6.4

**Comment by pratiman-91** [comment]
Try to use matplotlib version 3.3.2.

**Comment by lukelbd** [comment]
Closing this for now as I can't reproduce. Also it might have been fixed by #251
